### ISSUE TITLE
backup to nextcloud initialized indefinitely

### ISSUE THREAD

**Issue by BlackYps** [stevesoltys/backup#333]
When choosing nextcloud as the backup location, it hangs on "Initializing backup location" indefinitely (I checked after several hours and saw no progress). In my nextcloud it created the .SeedVaultAndroidBackup folder with a <randomHex>.sv folder inside. Inside that are a bunch of folders (00, 0a, 0b, etc) and then it seemingly stopped to do anything.
I made a local backup and after that it also worked for nextcloud. Interestingly it used a different folder structure now. it is <randomNumbers> and inside are folders for the backed up apps.
I hope this helps you, because now that it works I can't reproduce the original issue anymore.

- Nextcloud Android App version 3.17.0
- Pixel 4a, CalyxOS 2.10.0 (Android 11)
- Seedvault version 11-2.3

**Comment by initstring** [comment]
I had encountered this occasionally on CalyxOS, as well. The fix for me was the same thing I mentioned here: https://github.com/seedvault-app/seedvault/issues/266#issuecomment-977532043

Basically, disabling battery optimisation for the Nextcloud app. I'm not sure if that will help you, but it's worth looking.

**Comment by grote** [comment]
nothing actionable here so far, sounds like a nextcloud issue, maybe slow connection, maybe nextcloud gets killed by the OS.

#317 would help with this like with many other nextcloud related issues. You can also use the DavX5 WebDAV integration instead.

**Comment by eighthave** [comment]
I think this issue should be reopened, I'm getting it now using Lineage for microG 19.1-20231006 on a Google Pixel 3a sargo.  First, I tried backing up to Nextcloud.  The Battery Optimization for the Nextcloud app was set to "Unrestricted" beforehand.  I left it for almost an hour.  I navigated to the Backup Settings screen from the Settings app, and it didn't say it any more.  I ran "Backup Now" and it said "backup failed".  This is on fast wifi on the same network as the Nextcloud server, which has an SSD.  So this was not caused by a slow connection, it should be 10-30MB/s

* Seedvault 12-3.0
* Nextcloud 3.26.0
